### ISSUE TITLE
RSS feed

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#1070]
Add RSS icon in front of the search results count - but not part of the h2.

Something like this - i.e. icon is hidden from screen reader and text is invisible on screen.

`<a href="..."><i class="..." aria-hidden="true"></i><span class="sr-only">RSS feed for this result</span></a>`

![minimum - rss on search results](https://cloud.githubusercontent.com/assets/6945691/10704343/1340cf32-798b-11e5-97e5-31a835817d36.png)

Icon is Streamline rss-feed-box - needs to be added to sul_styles.
![rss-feed-box](https://cloud.githubusercontent.com/assets/6945691/10704542/598ac5be-798c-11e5-8046-6a7ec4d40744.png)

Color is $color-pantone-144 #e98300;

Bonus: add date-cataloged as an option to the sort dropdown.

![sort by acquisition date](https://cloud.githubusercontent.com/assets/6945691/10704348/1ee63d7c-798b-11e5-8d55-61a1ec15e323.png)

**Comment by jkeck** [comment]
@lmcglohon it looks like `date_cataloged` is not indexed (looking at the schema.xml on production solr).

Since we need to sort on this value we'll need it to be indexed, otherwise we'll get an error `can not sort on a field which is neither indexed nor has doc values: date_cataloged`

**Comment by lmcglohon** [comment]
@jkeck Updated code (https://github.com/sul-dlss/solrmarc-sw/pull/90) and am reindexing prior to pushing to production today so it will be indexed and available once the new production is in place

**Comment by jkeck** [comment]
:metal:
